ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1422,6 +1422,7 @@ class NoteTextComponent extends React.Component {
 		const paddingTop = 14;
 
 		const editorStyle = {
+			marginTop: this.state.noteTags.length === 0 && tagStyle.marginBottom,
```

Comment:
I think this: `this.state.noteTags.length === 0 && tagStyle.marginBottom` would evaluate to a boolean, wouldn't it? Probably we should instead set the margin explicitly, maybe with `!this.state.noteTags.length ? tagStyle.marginBottom : 0`

Reply:
No it would not. it means that if `this.state.noteTags.length === 0` is true, then apply `tagStyle.marginBottom`. Else do nothing.

Reply:
Hey @laurent22, so way this would work is,

If `this.state.noteTags.length === 0` is `true` then the value of `tagStyle.marginBottom` will be assigned to `marginTop`

Reply:
@Abijeet thanks for repeating. One more thing @laurent22. When `this.state.noteTags.length === 0` is `false`, `marginTop` is not rendered at all, no 0. The style will not exist. It is cleaner than leaving redundant `0`.